src/controllers/artist.js

```diff
@@ -0,0 +1,95 @@
+const db = require('../db/index.js');
+const createArtist = async (req, res) => {
+  const { name, genre } = req.body;
+
+  try {
+    const {
+      rows: [artist],
+    } = await db.query(
+      'INSERT INTO Artists (name, genre) VALUES ($1, $2) RETURNING *',
+      [name, genre]
+    );
+    res.status(201).json(artist);
+  } catch (err) {
+    console.log(err);
```

Comment:
As before could you remove the console logs since youre already returning something in the response?